### PR TITLE
Merge from dev to main (fix issues #234, #222 & email validation)

### PR DESCRIPTION
**Pull request by MuathJ**
#234 remove LinkedIn field
#222 make the "Join our Discord" text clickable
#--- Fix email validation when the domain extension is longer than 4 chars

Check changes here: https://dev--dorgv2.netlify.app/

### CODE REVIEW

File: src/components/FooterDesktop.tsx [hunk 1]
```diff
@@ -46,6 +46,18 @@ const FooterText = styled(Typography)({
   color: theme.palette.text.primary
 });
 
+const FooterLink = styled(Link)({
```
Comment (MatthiasBakken) [inline src/components/FooterDesktop.tsx:49]: did you want these style changes in the same PR?

File: src/constants/contactForm.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const SUBMIT_BUTTON_IMG_PATH = "/imgs/circled-arrow-icon.svg";
 export const MAX_NAME_LENGTH = 100;
 export const MAX_EMAIL_LENGTH = 100;
 export const MAX_MESSAGE_LENGTH = 6000;
-export const EMAIL_PATTERN = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i;
+export const EMAIL_PATTERN = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
```
Comment (MatthiasBakken) [inline src/constants/contactForm.ts:16]: Do you even need the following '{2,}' any longer? Not sure what you are matching with this? :)

**Review by MatthiasBakken — CHANGES_REQUESTED** [review]
Everything looks fine for the most part, curious about the last piece of the regex as I think it may not do anything. But I am in no way a regex master. See my comments. 👍

File: src/components/FooterDesktop.tsx (hunk 1 above)
Comment (MuathJ) [reply src/components/FooterDesktop.tsx:49]: This one is for this issue: https://github.com/dOrgTech/landing-page-v2/issues/222

File: src/constants/contactForm.ts (hunk 2 above)
Comment (MatthiasBakken) [reply src/constants/contactForm.ts:16]: I see what it is doing. It is matching 2 of anything, is that correct?

Comment (MuathJ) [reply src/constants/contactForm.ts:16]: Domain extensions are 2 char or longer, 1 char is invalid, I think it's fine to have it like this.

Comment (MuathJ) [reply src/constants/contactForm.ts:16]: > I see what it is doing. It is matching 2 of anything, is that correct?

Yes, 2 or more of any char for the domain extension

Comment (MatthiasBakken) [reply src/constants/contactForm.ts:16]: Perfect. I did some tests, it works great!

**Review by MatthiasBakken — APPROVED** [review]

**Pull request merged**